.rubocop_todo.yml

```diff
@@ -1,21 +1,20 @@
 # This configuration was generated by
 # `rubocop --auto-gen-config`
-# on 2022-07-07 21:26:21 UTC using RuboCop version 1.31.0.
+# on 2022-07-19 18:21:09 UTC using RuboCop version 1.31.2.
 # The point is for the user to remove these configuration records
 # one by one as the offenses are removed from the code base.
 # Note that changes in the inspected code, or installation of new
 # versions of RuboCop, may require this file to be generated again.
 
-
 # Offense count: 1
 # Configuration parameters: CountComments, CountAsOne.
 Metrics/ClassLength:
-  Max: 122
+  Max: 118
 
 # Offense count: 1
 # Configuration parameters: ExpectMatchingDefinition, CheckDefinitionPathHierarchy, CheckDefinitionPathHierarchyRoots, Regex, IgnoreExecutableScripts, AllowedAcronyms.
 # CheckDefinitionPathHierarchyRoots: lib, spec, test, src
 # AllowedAcronyms: CLI, DSL, ACL, API, ASCII, CPU, CSS, DNS, EOF, GUID, HTML, HTTP, HTTPS, ID, IP, JSON, LHS, QPS, RAM, RHS, RPC, SLA, SMTP, SQL, SSH, TCP, TLS, TTL, UDP, UI, UID, UUID, URI, URL, UTF8, VM, XML, XMPP, XSRF, XSS
 Naming/FileName:
   Exclude:
-    - 'lib/assembly-objectfile.rb'
\ No newline at end of file
+    - 'lib/assembly-objectfile.rb'
```

Comment:
This should really be in `.rubocop.yml` not `.rubocop_todo.yml`.

Reply:
good point - thanks.